code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -225,6 +225,23 @@
 							/obj/machinery/vending/cigarette = "ShadyCigs Deluxe",
 							/obj/machinery/vending/games = "\improper Good Clean Fun",
 							/obj/machinery/vending/autodrobe = "AutoDrobe",
+							/obj/machinery/vending/wardrobe/sec_wardrobe = "SecDrobe",
+							/obj/machinery/vending/wardrobe/medi_wardrobe = "MediDrobe",
+							/obj/machinery/vending/wardrobe/engi_wardrobe = "EngiDrobe",
+							/obj/machinery/vending/wardrobe/atmos_wardrobe = "AtmosDrobe",
+							/obj/machinery/vending/wardrobe/cargo_wardrobe = "CargoDrobe",
+							/obj/machinery/vending/wardrobe/robo_wardrobe = "RoboDrobe",
+							/obj/machinery/vending/wardrobe/science_wardrobe = "SciDrobe",
+							/obj/machinery/vending/wardrobe/hydro_wardrobe = "HyDrobe",
+							/obj/machinery/vending/wardrobe/curator_wardrobe = "CuraDrobe",
+							/obj/machinery/vending/wardrobe/bar_wardrobe = "BarDrobe",
+							/obj/machinery/vending/wardrobe/chef_wardrobe = "ChefDrobe",
+							/obj/machinery/vending/wardrobe/jani_wardrobe = "JaniDrobe",
+							/obj/machinery/vending/wardrobe/law_wardrobe = "LawDrobe",
+							/obj/machinery/vending/wardrobe/chap_wardrobe = "ChapDrobe",
+							/obj/machinery/vending/wardrobe/chem_wardrobe = "ChemDrobe",
+							/obj/machinery/vending/wardrobe/gene_wardrobe = "GeneDrobe",
+							/obj/machinery/vending/wardrobe/viro_wardrobe = "ViroDrobe",
```

Comment:
It's unnecessary in this pr but this list really should be generated

Reply:
What exactly do you mean by generated?

Reply:
The list should not have to be manually updated and just generate the list from the type path

Reply:
Mmm, it alright if I leave that for another PR?

Reply:
>It's unnecessary in this pr but this list really should be generated